runtime/libpgmath/test/lit.cfg

```diff
@@ -78,6 +78,9 @@ if config.architecture == "x86_64":
     elif config.operating_system == 'Linux':
         config.test_cflags += " -DTARGET_LINUX_X8664"
 
+if config.operating_system == 'Windows':
+    config.test_cflags += " -DTARGET_WIN"
+
```

Comment:
There is a check for windows above (line 70) where the function append_dynamic_library_path is called. Can this flag addition be moved there?